tests/js-tests/src/EffectsTest/EffectsTest.js

```diff
@@ -56,13 +57,27 @@ var EffecstsBaseLayer = BaseTestLayer.extend({
        this._super();
 
         var node = new cc.Node();
-	    var nodeGrid = new cc.NodeGrid();
+        node.setPosition(-winSize.width/2,0);
+        //if true
+        var visiablesize = director.getVisibleSize();
+        var gridRect = new cc.rect(visiablesize.width*0.1,
+                                   visiablesize.height*0.1,
+                                   visiablesize.width*0.5,
+                                   visiablesize.height*0.5);
+        //var actionGrid = new cc.GridAction();
+
+        var nodeGrid = cc.NodeGrid.create(gridRect); 
```

Comment:
Use `new cc.NodeGrid` instead of `cc.NodeGrid.create`, create function is deprecated